dashboard/config/scripts/levels/Allthethings Java Lab Playground.level

```diff
@@ -2,15 +2,67 @@
   <config><![CDATA[{
   "properties": {
     "encrypted": "false",
-    "long_instructions": "This is a sample Playground level. For now, it just looks and acts like Theater.",
+    "long_instructions": "This is a sample Playground level. It responds to clicks on the images that are in the program.",
     "mini_rubric": "false",
     "hide_share_and_remix": "false",
-    "csa_view_mode": "playground"
+    "csa_view_mode": "playground",
+    "submittable": "false",
+    "start_sources": {
+      "Dice.java": {
+        "text": "import org.code.playground.*;\nimport java.io.FileNotFoundException;\n\n// LESSON 1\npublic class Dice {\n\n  private ImageItem[] diceImages;\n\n  public Dice(String[] diceFiles, int xPos, int yPos) throws FileNotFoundException {\n    diceImages = new ImageItem[6];\n    \n\tfor (int i = 0; i < diceFiles.length; i++) {\n      diceImages[i] = new ImageItem(diceFiles[i], xPos, yPos,  50, 50);\n    }\n  }\n\n  public void displayDice(int value) {\n    Playground.board.addItem(diceImages[value - 1]);\n  }\n\n  public ImageItem getDice(int value) {\n    return this.diceImages[value - 1];\n  }\n\n  public void hideDice() {\n    for (ImageItem diceImage : this.diceImages) {\n    \tPlayground.board.removeItem(diceImage);\n    }\n  }\n}",
```

Comment:
We had a couple changes to the api we'll need to reflect here:
- we got rid of `addItem` and replaced it with `addTextItem` and `addImageItem`
- `board.exit()`now throws a `PlaygroundException`

Reply:
On that note, for my understanding, how do we update these files when we have API changes? Do we need to manually copy the code into this level file, or is this autogenerated somehow?

Reply:
it's manual :(